src/panels/config/info/system-health-card.ts

```diff
@@ -70,15 +76,30 @@ class SystemHealthCard extends LitElement {
           );
         }
         sections.push(html`
-          <table>
+          <table class="table-${domain}">
```

Comment:
Let's not do this with a class, also, I think it is not clear for a user that only this table will be copied?

Reply:
Originally I had copied everything, but it looked a bit weird with the "Lovelace" title in the middle of the copied data, especially since the first copied table (the systemhealth data) does not get any header/title since that is not actually part of the card content. 

I can change it back though if you prefer to copy everything in the card.

Reply:
We either should move the copy button to the table that will be copied, so it is clear what you are going to copy, or we should copy everything.

Reply:
The I would probably prefer to copy all. If the user only needs part of the data, removing it is faster than copying manually each table.

Is there a reason why the card headers are not actual `<h>` tags? Then when copying e.g. into a word document they would be correctly formatted (currently only the header "Lovelace" is).

Reply:
No, we should move to using `h` tags more.

Reply:
I will provide a separate PR to make `<ha-card>` use a `<h>` tag for the header. As a result this PR is ready and will read the complete card now for the system info.

Reply:
On second thought: The ha-card change should happen first, since afterwards I also need to switch out <div> against <h1> for the system health info => converting to draft for now.